pelicun/tests/basic/test_uq.py

```diff
@@ -1114,6 +1172,36 @@ def test_LogNormalRandomVariable_cdf() -> None:
     assert np.allclose(cdf, (0.0, 0.0, 0.2441086, 0.5, 0.7558914), rtol=1e-5)
 
 
+def test_LogNormalRandomVariable_variable_theta_cdf() -> None:
+    rv = uq.NormalRandomVariable(
```

Comment:
Why is this calling `NormalRandomVariable` when we're testing the lognormal one? Same question for lines 1192 and 1244.